health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/DraggableField.js

```
@@ -47,8 +47,19 @@ function DraggableField({
 
   drag(drop(ref));
   return (
-    <div ref={ref} style={{ opacity: isDragging ? 0.5 : 1, display: "flex", alignItems: "center" }}>
-      <span style={{ cursor: "grab", marginRight: "8px", lineHeight: "1" }}>⋮</span>
+    <div className="draggableField-cont" ref={ref} style={{ opacity: isDragging ? 0.5 : 1, display: "flex", alignItems: "center" }}>
+      {/* <span style={{ cursor: "grab", marginRight: "0.5rem", marginLeft: "0.5rem", lineHeight: "1" }}>⣿</span> */}
+      <div class="drag-handle">
+        <span></span>
+        <span></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)